### PR TITLE
Added baylands model and example world

### PR DESCRIPTION
**Pull request by ahcorde**
Added baylands 
 - model
 - example world

baylands

co-author: @tfoote

### CODE REVIEW

**Comment by tfoote** [comment]
Please test this out as is for feedback.  Before it's ready to merge we're going to move the models to be hosted at gazebosim.org/models so that the model data isn't stored here in this repository. But before we finalize the model upload there we'd like to get any feedback from reviews here.

**Comment by Jaeyoung-Lim** [comment]
@tfoote @ahcorde This is extremely cool!

These worlds would be really useful for testing different functionalities in the flight controllers such as terrain following, obstacle avoidance.

a. I was not able to spawn the model properly. The gazebo segfaults when trying to run this model. Are there any minimum requirements regarding gazebo versions? 
```
[ 3365.113498] gzclient[17944]: segfault at 120 ip 00007f50cb45efa0 sp 00007f501d15f8a8 error 4 in libpthread-2.27.so[7f50cb455000+1a000]
```
Currently I am running gazebo v9.0.0 on Ubuntu 18.04 Bionic

b. Would it be possible to get the GPS coordinates of the origins? That way we can also align the GPS and show the map accordingly on QGroundControl

**Comment by ahcorde** [comment]
You need to use a gazebo version which support heighmaps. I tested with Gazebo 9 and 11. World are quite big, maybe a decent GPU.

I have all the gps origins, where I should include it?

**Comment by mrivi** [comment]
I tested this and I was able to load the worlds however it takes forever, around 3 minutes

**Comment by Jaeyoung-Lim** [comment]
@ahcorde I see that the models have not been added to the model database. Are there any plans to add the models in the database anytime soon?

**Comment by ahcorde** [comment]
The PR to upload the models is still open https://bitbucket.org/osrf/gazebo_models/pull-requests/360

**Comment by Jaeyoung-Lim** [comment]
@ahcorde On the gps origins, I am trying to come up with a way where the gps origin of the world is defined in the world file as in https://github.com/PX4/sitl_gazebo/pull/441

Once we get this cleared, it would be great if we can get the gps coordinates of those large models so we can match them with the groundstation map UI

**Comment by Jaeyoung-Lim** [comment]
@ahcorde How can we coordinate on getting the gps origins of those worlds? 

This will be simply defining `spherical_coordinates` as done in #441

**Comment by ahcorde** [comment]
```
    os.environ["PX4_HOME_LAT"] = "37.413534";
    os.environ["PX4_HOME_LON"] = "-121.996561";
    os.environ["PX4_HOME_ALT"] = "1.3";
```

**Comment by TSC21** [comment]
> ```
>     os.environ["PX4_HOME_LAT"] = "37.413534";
>     os.environ["PX4_HOME_LON"] = "-121.996561";
>     os.environ["PX4_HOME_ALT"] = "1.3";
> ```

These should be added to the world file itself with the spherical_coordinates tag. Although I would assume this needs to be on the Gazebo model database instead.

**Comment by Jaeyoung-Lim** [comment]
@ahcorde Awesome, thanks. Do you have a ETA for this model to be deployed in the model database?

**Comment by ahcorde** [comment]
You can see in this other repo how this [code](https://github.com/osrf/uav_testing/blob/ros2/yosemite_valley/launch/drone_yosemite.launch.py) is being used

**Comment by ahcorde** [comment]
I don't know the meaning of ETA.

**Comment by TSC21** [comment]
> You can see in this other repo how this [code](https://github.com/osrf/uav_testing/blob/ros2/yosemite_valley/launch/drone_yosemite.launch.py) is being used

We won't be using launch files. As @Jaeyoung-Lim said, the logic way of doing it is setting the LLA directly in the world file.

> I don't know the meaning of ETA.

What is the time estimate to have them available in the database? (Estimated Time of Arrival/Available)

**Comment by ahcorde** [comment]
@tfoote ping

File: worlds/baylands.world [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <clouds>
+          <speed>12</speed>
+        </clouds>
+      </sky>
+    </scene>
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+    </include>
+
```
Comment (Jaeyoung-Lim) [inline worlds/baylands.world:15]: Could you define the gps origin as defined in https://github.com/PX4/sitl_gazebo/blob/master/worlds/sonoma_raceway.world#L12 ?

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]
Thanks for getting it merged in the model database.
Could you squash the commits so that large files don't get merged in the repo?

Thanks!

**Review by Jaeyoung-Lim — APPROVED** [review]
Looks good on my side, thanks!

I don't see the models in http://models.gazebosim.org/ yet. Do you know when the model will be deployed?

Would be better if we merge once the models are available in the model database

**Comment by Jaeyoung-Lim** [comment]
It is now in the model database and looks pretty good

[Demo](https://youtu.be/ir1fwOLcd4E)

Thanks @ahcorde !

**Review by TSC21 — APPROVED** [review]

**Comment by TSC21** [comment]
@hamishwillee good for docs :) we should document what worlds we have available.

**Comment by hamishwillee** [comment]
@TSC21 @Jaeyoung-Lim Re https://github.com/PX4/sitl_gazebo/pull/405#issuecomment-604161789

That would be nice; who would like to volunteer?
This does not require my specific expertise, though of course I am very happy to review.

I think this might be worth adding this as a sub page of Gazebo. Along with splitting out the vehicle models into their own pages. - e.g. http://dev.px4.io/master/en/simulation/gazebo.html#quadrotor

Thoughts?

**Comment by Jaeyoung-Lim** [comment]
@hamishwillee I will write it down! I think it makes sense to separate the pages since the page is getting a bit long now

**Comment by hamishwillee** [comment]
Great. FYI, started the process https://github.com/PX4/Devguide/pull/995

**Pull request merged**